tests/Integration/Behaviour/Features/Context/Domain/Product/CommonProductFeatureContext.php

```diff
@@ -129,6 +129,14 @@ public function assertProductFields(string $productReference, TableNode $table)
         $data = $table->getRowsHash();
 
         $this->assertBoolProperty($productForEditing, $data, 'active');
+        $this->assertStringProperty($productForEditing, $data, 'visibility');
```

Comment:
Mmm, it's weird I believe these checks were removed from the Common feature to the `UpdateOptionsFeatureContext` in this PR https://github.com/PrestaShop/PrestaShop/pull/20554
Probably a rebase that reintroduced them

The idea that you started with other sub domains was to have a context dedicated to each product sub domains: SEO, Options, Shipping, ... Because the common context was getting bigger and bigger, so it should only deal with the "general" fields (only active seemed independent 😅 )

So in this idea `redirect_type` should be checked in the new `UpdateSeoFeatureContext` you introduced